css/main.scss

```diff
@@ -160,32 +160,67 @@ header {
     text-align: right;
     padding-right: 20px;
 
-    .nav {
-      // neat animations
-      &:before, &:after {
-        content: '';
-        width: 0;
-        height: 0;
-        display: block;
-        background-color: white;
+  }
 
-        transition: width 0.2s;
-      }
+  .dropdown {
+    overflow: hidden;
+  }
```

Comment:
```suggestion
```

This fixes your alignment problem, but I don't know what implications it will have for mobile

Reply:
Ok thanks - I'll give it a go.